pluginDefinition.json

```diff
@@ -14,8 +14,8 @@
     "descriptionDefault": "Zowe Editor",
     "isSingleWindowApp": true,
     "defaultWindowStyle": {
-      "width": 1600,
-      "height": 800
+      "width": 1200,
```

Comment:
Makes sense. Gone will be the days of muscle memory opening Editor and instantly minimizing 20% of it.